ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -546,13 +560,17 @@ long getErrorCount() {
     private final AtomicBoolean instanceConfigInitialized = new AtomicBoolean(false);
     private final AtomicBoolean liveStateInitialized = new AtomicBoolean(false);
     private final AtomicBoolean idealStateInitialized = new AtomicBoolean(false);
+    private final AtomicBoolean routingTableInitialized = new AtomicBoolean(false);
```

Comment:
Don't need a second variable for this, you can just check that `routingTableInitLatch.getCount() == 0`. This should have roughly equivalent performance since it is just checking the value of a `volatile int`

Reply:
fair point, will remove this to make it simple.